echopype/convert/ek80.py

```diff
@@ -545,6 +546,31 @@ def _export_zarr(self, save_settings, file_idx=0):
         else:
             self._set_groups(raw_file, out_file, save_settings=save_settings)
 
+    def _copy_group(self, src_file, trg_file):
+        # Utility function for copying the filter coefficients from one file into another
+        src = netCDF4.Dataset(src_file)
+        trg = netCDF4.Dataset(trg_file, mode='a')
```

Comment:
Could you check if the saved data variables can be read correctly using xarray? I am also wondering why you choose to use netCDF4 here and not xarray?

Reply:
xarray does not handle complex numbers